docs/docker-stack/changelog.rst

```diff
@@ -302,6 +302,17 @@ here so that users affected can find the reason for the changes.
 +--------------+---------------------+-----------------------------------------+------------------------+------------------------------------------------+
 | Date         | Affected images     | Potentially breaking change             | Reason                 | Link to Pull Request / Issue                   |
 +==============+=====================+=========================================+========================+================================================+
+| 02 Aug 2024  | 2.9.3               | * The ``apache-airflow-providers-fab``  | FAB provider logout    | https://github.com/apache/airflow/issues/40922 |
+|              |                     |   upgraded from 1.1.1 to 1.1.2          | did not work for 2.9.3 |                                                |
```

Comment:
It's a small thing, but I think it's `1.2.2` instead of `1.1.2`.

Reply:
Not small.  very important. What a typo 🤦  - it also would went to CVE announcement if you did not see it - thanks for spotting it. Fix coming in #41215